bot/exts/moderation/infraction/management.py

```diff
@@ -272,7 +274,7 @@ async def search_reason(self, ctx: Context, reason: str) -> None:
     async def search_by_actor(
         self,
         ctx: Context,
-        actor: t.Union[discord.Member, t.Literal["m", "me"]],
+        actor: t.Union[t.Literal["m", "me"], UnambiguousMember],
```

Comment:
I just realized, the actor doesn't necessarily have to be a member.

Reply:
For some reason I thought it did despite thinking this over at least a dozen times during development. Thanks.

Reply:
Done :+1: